lib/index.js

```diff
@@ -13,3 +13,6 @@ for(const {contexts} of contextModules) {
     jsonLdDocumentLoader.addStatic(contextUrl, context);
   }
 }
+
+const {constants} = vcRevocationListContext;
+export {constants};
```

Comment:
I think this can be a one-liner?
```suggestion
export const {constants} = vcRevocationListContext;
```